lang_GENERIC_base/AST_generic.ml

```diff
@@ -140,6 +140,23 @@
  * See also pfff/lang_GENERIC/
 *)
 
+(*****************************************************************************)
+(* Accessories *)
+(*****************************************************************************)
+
+(* A set of metavariables. Access cost is O(log n). *)
+module String_set = struct
+  include Set.Make (String)
+
+  type string_list = string list
+  [@@deriving show]
+
+  let pp fmt x =
+    pp_string_list fmt (elements x)
+end
+
+type string_set = String_set.t
```

Comment:
I have a defunctorized set in pfff/commons/set_.ml so you can directly use 'string Set_t.t', like you use a list,
so no need for those functors.

Reply:
oh nice.